MMM-Fuel.js

```diff
@@ -242,6 +242,37 @@ Module.register("MMM-Fuel", {
         var row = document.createElement("tr");
 
         var station_name = data.name;
+		if(station_name=="Hilden, Herder Str. 44"){
```

Comment:
I will not merge this static if conditions into the module, because it will probably only affect the gas stations for you especially those with street names. But I could live with a generic solution like:

config: {
  replaceNames: {
    "HILDEN, HOCHDAHLER STR": "Shell",
    ...
  }
}
so everyone whos interested can define it for himself. in the code you would check
if(this.config.replaceNames.hasOwnProperty(data.name)){
//replace string
}